scratch-debugger/debug.sh

```diff
@@ -22,6 +22,7 @@ set -o pipefail
 TMP_SUBDIR="${TMP_SUBDIR:-debug-tools}"
 CONTEXT="${KUBECONTEXT:-}"
 ARCH="${ARCH:-amd64}"
+DOCKER_DOWNLOAD_URL="${DOCKER_DOWNLOAD_URL-https://download.docker.com/linux/static/stable/x86_64/docker-17.12.0-ce.tgz}"
```

Comment:
nit: use `${VAR:-default}` (`:-` vs `-`) syntax, for consistency

Reply:
yeah this is a typo, good catch!